clarity-ext-scripts/clarity_ext_scripts/covid/parse_pcr_execution.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+import pandas as pd
+from datetime import datetime
+
+CT_HEADER = u"Cт"
+
+
+class ParsePcrExecution(object):
+    def __init__(self, context):
+        self.context = context
+
+    def execute(self):
+        file_name = "Result file"
+        f = self.context.local_shared_file(file_name, mode="rb")
+        data = pd.read_excel(f, 'Results', index_col=0, skiprows=7, encoding='utf-8')
```

Comment:
Later on we need to make the file format reading configurable here, e.g. skipping the first 7 rows is probably not ok depending on the output file format. But for now we should focus on handling the Quant Studio 7 machines.